### PR TITLE
moviesDescription page added

### PR DESCRIPTION
**Pull request by ashishprodev**

### CODE REVIEW

File: src/screen/MoviesDescription.js [hunk 1]
```diff
@@ -1,21 +1,170 @@
...
+} from 'react-native';
 import React from 'react';
 import {colors} from '../constants';
 
 const MoviesDescription = () => {
   return (
-    <View style={styles.MoviesDescription}>
-      <Text>MoviesDescription</Text>
-    </View>
+    <SafeAreaView style={{blackground: colors.black}}>
```
Comment (hangingPandaReact) [inline src/screen/MoviesDescription.js:15]: inline css has been used remove it

File: src/screen/MoviesDescription.js [hunk 2]
```diff
@@ -1,21 +1,170 @@
...
+      <ImageBackground
+        source={{
```
Comment (hangingPandaReact) [inline src/screen/MoviesDescription.js:17]: Covert these thing into component

File: src/screen/MoviesDescription.js [hunk 3]
```diff
@@ -1,21 +1,170 @@
...
+          <Image
+            style={styles.poster}
+            source={{
+              uri: 'https://i0.wp.com/urbanasian.com/wp-content/uploads/2021/12/20211220_132533__01.jpg?w=678&ssl=1',
+            }}
+          />
+          <View style={styles.details}>
+            <Text style={styles.moviesName}>The Kashmir Files</Text>
+            <Text style={styles.Catagories}>History ○ Thriller</Text>
+            <Text style={styles.Country}>India</Text>
```
Comment (hangingPandaReact) [inline src/screen/MoviesDescription.js:32]: these text should be a component and things should be used only component

**Review by hangingPandaReact — CHANGES_REQUESTED** [review]

File: App.js [hunk 4]
```diff
@@ -1,15 +1,25 @@
...
+import {NavigationContainer} from '@react-navigation/native';
 
 const App = () => {
-  console.log('hello dev');
+  const Stack = createNativeStackNavigator();
   return (
     <Provider store={store}>
       <HomeScreen />
-      {/* navigate to MoviesDescription */}
+      {/* <NavigationContainer>
```
[the remark this answers is not in the document]
Comment (ashishprodev) [reply App.js]: okay sir

File: src/component/card/Card.js [hunk 5]
```diff
@@ -14,19 +15,24 @@ import Slice, {movieSlice} from '../../redux/slice/Slice';
...
           style={styles.cardContiner}
         />
       </TouchableWithoutFeedback>
-      <TouchableOpacity style={styles.details}>
-        <Text style={styles.moviesName}>{item.Title}</Text>
-        <Text style={styles.Rating}>{item.Rated}</Text>
-        <Text style={styles.year}>{item.Year}</Text>
+      <TouchableOpacity>
+        <View style={styles.detail}>
+          <Text style={styles.moviesName}>{item.Title}</Text>
```
[the remark this answers is not in the document]
Comment (ashishprodev) [reply src/component/card/Card.js:31]: okay sir, i will change

File: src/screen/MoviesDescription.js (hunk 3 above)
Comment (ashishprodev) [reply src/screen/MoviesDescription.js]: okay sir, i will wrap into details container

File: src/screen/MoviesDescription.js (hunk 2 above)
Comment (ashishprodev) [reply src/screen/MoviesDescription.js:19]: okay sir

File: src/screen/MoviesDescription.js (hunk 1 above)
Comment (ashishprodev) [reply src/screen/MoviesDescription.js]: got it sir, i will change

**Pull request merged**